templates/projects.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+
+{% block meta_title %}Projects{% endblock meta_title %}
+
+{% block main_content %}
+
+    <h4>Projects list:</h4>
+
+    <table class="table table-hover table-striped">
+        <thead>
+            <tr>
+                <th> id </th>
+                <th> title </th>
+                <th> members number </th>
+                <th> members list </th>
+                <th> start date </th>
+                <th> end date </th>
+        </thead>
+        <tbody>
+        {% for project in projects %}
+
+            <tr>
+                <td>{{ project.pk }}</td>
+                <td><a href="{% url "project_detail" project.pk %}">{{ project.title }}</a></td>
+                <td>{{ project.relation.all.count }}</td>
```

Comment:
you call relation here, but you haven't prefetched it. It will cause tons of separate queries if you have multiple projects